### PR TITLE
fix(core-translate): translations for password control

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
MANAGER-3036

Translations added for the oui-password control

### CODE REVIEW

File: packages/manager/modules/core/src/translate/ovh-ui-angular/index.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ angular.module(moduleName, [
     ouiNavbarConfiguration,
     ouiPaginationConfiguration,
     ouiStepperConfiguration,
+    ouiPasswordConfiguration,
```
Comment (antleblanc) [inline packages/manager/modules/core/src/translate/ovh-ui-angular/index.js]: Could be sorted;)

File: packages/manager/modules/core/src/translate/ovh-ui-angular/index.js [hunk 2]
```diff
@@ -121,6 +122,19 @@ angular.module(moduleName, [
           submitButtonLabel: $translate.instant('common_stepper_submit_button_label'),
           cancelButtonLabel: $translate.instant('common_stepper_cancel_button_label'),
         });
+
+        set(ouiPasswordConfiguration, 'translations', {
```
Comment (antleblanc) [inline packages/manager/modules/core/src/translate/ovh-ui-angular/index.js]: Could be placed right after `ouiPaginationConfiguration`;)

File: packages/manager/modules/core/src/translate/ovh-ui-angular/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -64,5 +64,15 @@
...
+  "common_file_removeFile": "Supprimer le fichier de la sélection",
+  "common_password_all_rules_met": "Toutes les règles de mot de passe sont respectées.",
+  "common_password_hide_password": "Masquer le mot de passe",
+  "common_password_show_password": "Montrer le mot de passe",
+  "common_password_valid_rule": "Règle valide.",
+  "common_password_invalid_rule": "Règle invalide.",
+  "common_password_risky_password": "Mot de passe risqué.",
+  "common_password_bad_password": "Mauvais mot de passe.",
+  "common_password_weak_password": "Mot de passe faible.",
+  "common_password_good_password": "Bon mot de passe.",
```
Comment (jleveugle) [inline packages/manager/modules/core/src/translate/ovh-ui-angular/translations/Messages_fr_FR.json]:
```suggestion
  "common_password_good_password": "Mot de passe correct.",
```

File: packages/manager/modules/core/src/translate/ovh-ui-angular/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -64,5 +64,15 @@
...
   "common_file_maxsizeError": "Le fichier dépasse la taille limite",
-  "common_file_removeFile": "Supprimer le fichier de la sélection"
+  "common_file_removeFile": "Supprimer le fichier de la sélection",
+  "common_password_all_rules_met": "Toutes les règles de mot de passe sont respectées.",
+  "common_password_hide_password": "Masquer le mot de passe",
+  "common_password_show_password": "Montrer le mot de passe",
+  "common_password_valid_rule": "Règle valide.",
+  "common_password_invalid_rule": "Règle invalide.",
+  "common_password_risky_password": "Mot de passe risqué.",
+  "common_password_bad_password": "Mauvais mot de passe.",
```
Comment (jleveugle) [inline packages/manager/modules/core/src/translate/ovh-ui-angular/translations/Messages_fr_FR.json]:
```suggestion
  "common_password_bad_password": "Mot de passe incorrect.",
```

File: packages/manager/modules/core/src/translate/ovh-ui-angular/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -64,5 +64,15 @@
   "common_file_fileSelector": "Sélectionnez un fichier",
   "common_file_filesSelector": "Sélectionner des fichiers…",
   "common_file_maxsizeError": "Le fichier dépasse la taille limite",
-  "common_file_removeFile": "Supprimer le fichier de la sélection"
+  "common_file_removeFile": "Supprimer le fichier de la sélection",
+  "common_password_all_rules_met": "Toutes les règles de mot de passe sont respectées.",
+  "common_password_hide_password": "Masquer le mot de passe",
+  "common_password_show_password": "Montrer le mot de passe",
```
Comment (jleveugle) [inline packages/manager/modules/core/src/translate/ovh-ui-angular/translations/Messages_fr_FR.json]:
```suggestion
  "common_password_show_password": "Afficher le mot de passe",
```

File: packages/manager/modules/core/src/translate/ovh-ui-angular/index.js (hunk 1 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/core/src/translate/ovh-ui-angular/index.js]: This is done :)

File: packages/manager/modules/core/src/translate/ovh-ui-angular/index.js (hunk 2 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/core/src/translate/ovh-ui-angular/index.js]: This is done :)

File: packages/manager/modules/core/src/translate/ovh-ui-angular/translations/Messages_fr_FR.json (hunk 3 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/core/src/translate/ovh-ui-angular/translations/Messages_fr_FR.json]: Corrected :)

File: packages/manager/modules/core/src/translate/ovh-ui-angular/translations/Messages_fr_FR.json (hunk 4 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/core/src/translate/ovh-ui-angular/translations/Messages_fr_FR.json]: Corrected :)

File: packages/manager/modules/core/src/translate/ovh-ui-angular/translations/Messages_fr_FR.json (hunk 5 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/core/src/translate/ovh-ui-angular/translations/Messages_fr_FR.json]: Corrected :)

**Pull request merged**